docs/Open_Source_Checklist.adoc

```diff
@@ -1,65 +1,130 @@
+
+== People & Maintenance
+
+* Must ensure that any maintaining team has received some education or experience regarding working with Open Source. For example, read through guides at https://opensource.guide.
+* Must ensure that any maintaining team has a plan for handling Merge/Pull-requests (code reviews, etc.).
+* Must ensure that any maintaining team has a plan for handling community issues (responding to issues, reviewing and merging pull requests).
+* Must ensure that any maintaining team has someone responsible for following up on security issues.
+* Must make sure that every project README has a `Maintainer` header, whether it's a team, an individual, or a role.
+
+* May discuss if there is a need for a release plan. If so, ensure it is clear how it should be announced and promoted.
+** Additionally, the project SHOULD use a CODEOWNERS file.
+ A CODEOWNERS file can provide more granularity in describing project maintenance, whereas the README’s Maintainer heading might be more general.
 
 == Project Quality
 
-* MUST verify that the project has undergone a code review.
-* SHOULD discuss and establish the project's testing goals and ambitions.
+* Must verify that the project has undergone a code review.
+* Should discuss and establish the project's testing goals and ambitions.
+* Should make it easy to use the project - documentation, examples, pre-built releases, etc.
 
 == Release and Versioning
 
-* SHOULD use semantic versioning and release tags
+* Should use semantic versioning and release tags.
 
-== Naming and Trademarks Check
+== Security
 
-* SHOULD ensure that the project name does not conflict with an existing project or infringe on trademarks
-** Conduct a general search engine check for the proposed project name.
-** Perform a https://www.prv.se/en/ip-professional/trademarks/trademark-databases/[Trademark Search]
-+
-NOTE: It might be perfectly acceptable to use a name reminiscent of an existing trademark if the existing trademark is used for other services/areas and is not recognized as a well-known trademark.
+This part of the checklist is based on the https://github.com/ossf/wg-best-practices-os-developers/blob/main/docs/Concise-Guide-for-Developing-More-Secure-Software.md[Open Source Security Foundation (OpenSSF) best practices], version: 2023-06-14, https://openssf.org/blog/2024/04/15/open-source-security-openssf-and-openjs-foundations-issue-alert-for-social-engineering-takeovers-of-open-source-projects/[(and companion post)].
 
-== Legal and Licensing
+=== General
 
-* MUST ensure that the project's license does not conflict with third-party licenses.
-* The project licence declarations MUST follow the https://reuse.software/[The REUSE licensing specification]
+* Must use two-factor authentication (2FA) or multifactor authentication (MFA) to make it harder for account takeovers.
```

Comment:
Do we mean here that we expect that the maintainers and contributors are using 2FA on their accounts? Is this something that a repository owner will be able to validate on its contributors? 🙂

Reply:
Good point!!- It depends.. - as long as the contribitor is part of an organisation it will be. Otherwise no. But GitHub is still solving that for us. https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/about-mandatory-two-factor-authentication by slowly making it mandatory.

However, even if it not enforcable in all cases, it should be in there as a practice and rule. There will be other rules that are not verifiable at all times in there.

Reply:
Ahh alright! Thanks for the link 😊